auth/auth.go

```diff
@@ -33,7 +32,7 @@ type Auth interface {
 	// Options set for auth
 	Options() Options
 	// Generate a new account
-	Generate(id string, opts ...GenerateOption) (*Account, error)
+	Generate(id, secret string, opts ...GenerateOption) (*Account, error)
```

Comment:
Leave secret as an option

Reply:
Why so? Without a secret the account would be useless since no tokens can be generated?

Reply:
Because I should be able to call Generate with an id and it sets me a unique secret. Its not a mandatory requirement since you get it back in account.